app/serializers/school_serializer.rb

```diff
@@ -0,0 +1,49 @@
+class SchoolSerializer < Blueprinter::Base
+  class AttributesSerializer < Blueprinter::Base
+    exclude :id
+
+    field :name
+    field(:urn) { |school, _| school.urn.to_s }
+    field(:cohort) do |_school, options|
+      options[:contract_period_id]&.to_s
+    end
+    field(:in_partnership) do |school, options|
+      in_partnership?(school, options)
+    end
+    field(:induction_programme_choice) do |school, options|
```

Comment:
Im not sure about passing in lead provider/contract period as options to the serialiser, can we instead surface them as attributes from the query to read here on the school?